uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -304,6 +304,26 @@ define([
       });
     });
 
+    describe('delete()', function () {
+      var idToDelete = '123';
+
+      beforeEach(function () {
+        spyOn(LeaveRequestAPI, 'sendGET');
```

Comment:
How is it different from other methods like `all()` and others. Or do you mean we should not test any api calls which does not change the data before sending it back to the callee.